README.rst

```diff
@@ -93,6 +96,204 @@ For WNS, you need both the ``WNS_PACKAGE_SECURITY_KEY`` and the ``WNS_SECRET_KEY
 - ``USER_MODEL``: Your user model of choice. Eg. ``myapp.User``. Defaults to ``settings.AUTH_USER_MODEL``.
 - ``UPDATE_ON_DUPLICATE_REG_ID``: Transform create of an existing Device (based on registration id) into a update. See below `Update of device with duplicate registration ID`_ for more details.
 
+**Browser settings**
+
+Install:
```

Comment:
It's a nitpick, but the sub-headings should be tabbed over so it is clear that they are for the Webpush feature.

Reply:
Ok

Reply:
Done [40921fa09cc382653900d92497456afa46c2521c]